docs/ota-client-guide/modules/ROOT/pages/build-quemu.adoc

```diff
@@ -1,3 +1,38 @@
-// MC: Included from repo in Gitlab: https://main.gitlab.in.here.com/olp/edge/ota/documentation/ota-connect-docs/tree/master/docs/getstarted/modules/ROOT/pages/qemuvirtualbox.adoc
+= Build a QEMU/VirtualBox 
```

Comment:
We are missing the word "image" here?

Reply:
Yep, I'll fix in a followup PR (this one's merged already)